async-development/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href='style.css' rel='stylesheet'>
+    <title>GetWeatherAPI</title>
+</head>
+<body>
+    <form id='form' method="GET" action="#">
```

Comment:
Remove unnecessary attributes
```suggestion
    <form id='form'>
```